### ISSUE TITLE
Spawn timers and spawn killing

### ISSUE THREAD

**Issue by Dakota0001** [Dakota0001/DakBots#11]
Reducing spawn timer to 3 seconds should help with unneeded heavy delays players may face in the gamemode when things go wrong, but what other things, if anything, should or could be done to prevent situations of players spawning at bad points? If map is working player can see that there is a vehicle with smoke/fire/explosions at a point they're trying to spawn in, would just fixing map on clients where it is broken be enough warning not to spawn in a potentially dangerous location?

**Comment by Stooberton** [comment]
I reduced the respawn time to 5 seconds in bfded2fe0cebf6a0fdba6850cdfb64b339b30d77, I figured we should try halfway first.

**Comment by Stooberton** [comment]
I think there's a lot of soul and just good fun getting to the brutal spawn camping stage. 

Normally spawnkilling would be bad, but there are multiple spawn locations to choose from as a player and there's a lot of bots for the enemy to deal with.

Additionally, it complements the balance of the game. To suppress a point well means investing points in it on a vehicle. Players shouldn't be trying to spawn vehicles or really do anything at a point is being suppressed, spawn somewhere else that is safe or flanking. On that note I would say that spawnkilling should be encouraged. The current meta is about suppressing the point until it can be captured. Trying to spawn a vehicle infront of someone for an honorable duel or whatever is just being silly.

**Comment by Dakota0001** [comment]
I agree with the concept of keeping spawn camping in, as it is mostly the core concept to capping being that you must suppress the cap, doing so heavily bleeds points from the opposing team and forces them to spawn elsewhere as players and attempt to stop your siege if they want to regain the point to spawn in safely, push forward, and prevent the bleed out. In the cases where it is the last cap point a team has left it is already over for that team anyway, maps are big and points are plentiful, if they have none they've made a series of tactical mistakes that has led up to this point and the massive point bleed just from cap disadvantage most often prevents it from even getting to the point of total spawncamp.
